OKR.md

```diff
@@ -34,3 +34,10 @@ Quarterly statement of OKRs (Objectives and Key Results) for the "IPFS in Web Br
 
 - `dweb:` proposal is documented
 - `Ways to deal/support the Content Origin Policy for IPFS links` are documented
+
+**Publish IPFS UI Style Guide**
```

Comment:
"Publish X" sounds like a Result, not an Objective. Perhaps something like: 
> Developers are empowered by IPFS Design/Style Guide and a growing library of reusable UI components

would be a better Objective?

Alternatively, we could just move this under existing "Improved initial experience for non-technical users of mainstream browsers" objective.

Reply:
I'll rework it

Reply:
>  move this under existing "Improved initial experience for non-technical users of mainstream browsers" objective.

Good idea, I've done just that.